pkg/kubectl/cmd/cmd.go

```diff
@@ -110,23 +110,26 @@ __custom_func() {
 }
 `
 	valid_resources = `Valid resource types include:
-   * pods (aka 'po')
-   * replicationcontrollers (aka 'rc')
+   * componentstatuses (aka 'cs')
```

Comment:
Looks like this list was previously in priority (how likely are you to want to look at these) order.  I'm inclined to keep it in that order unless alphabetical was spec'ed somewhere else.